hack/update-generated-crd.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -euo pipefail
+
+OUTDIR="${OUTDIR:-}"
+if [ -z "$OUTDIR" ]; then
+  OUTDIR="$(mktemp -d)"
+fi
+
+SKIP_COPY="${SKIP_COPY:-}"
+
+# Generating against the github.com/openshift/api/operator/v1 will currently
+# generate some failures until other types' markers and structures are updated
+# to be compatible with the latest metadata expectations. For now, generate the
+# minimal set of type files we know are needed for the ingress types.
+set -x
+GO111MODULE=on GOFLAGS=-mod=vendor go run sigs.k8s.io/controller-tools/cmd/controller-gen crd:trivialVersions=true \
+  paths=./vendor/github.com/openshift/api/operator/v1/doc.go\;./vendor/github.com/openshift/api/operator/v1/types.go\;./vendor/github.com/openshift/api/operator/v1/types_ingress.go \
+  output:crd:dir="$OUTDIR"
+set +x
+
+if [ -z "$SKIP_COPY" ]; then
+  cp $OUTDIR/operator.openshift.io_ingresscontrollers.yaml manifests/00-custom-resource-definition.yaml
```

Comment:
Better to quote to avoid word splitting on `$OUTDIR`.

Reply:
Fixed

Reply:
Line 5 has the same issue,

Reply:
Already fixed